plugin/src/main/kotlin/com/jetbrains/packagesearch/plugin/ui/PackageSearchPackagePanel.kt

```diff
@@ -16,19 +26,29 @@ fun PackageSearchPackagePanel(
     onLinkClick: (String) -> Unit,
     onPackageEvent: (PackageListItemEvent) -> Unit,
 ) {
-    HorizontalSplitLayout(
-        first = { PackageSearchModulesTree(it, onSelectionModulesSelectionChanged) },
-        second = {
+    val splitPaneState = rememberSplitPaneState(PackageSearchMetrics.Splitpane.firstSplitterPositionPercentage )
```

Comment:
Save such values in the view model of this composable. This way, it will survive opening and closing of the toolwindow.